proposals/function-pointers.md

```diff
@@ -63,174 +64,173 @@ unsafe class Example {
+    public static void Log() { }
 
     void Use() {
-        func* void() ptr1 = &Util.Log;
+        delegate*<void> ptr1 = &Util.Log;
 
-        // Error: type "func* void()" not compatible with "func int()";
-        func* int() ptr2 = &Util.Log; 
+        // Error: type "delegate*<void>" not compatible with "delegate*<int>";
+        delegate*<int> ptr2 = &Util.Log;
 
         // Okay. Conversion to void* is always allowed.
         void* v = &Util.Log;
    }
 }
 ```
 
-The conversion of an address-of method group to `func*` has roughly the same process as method group to `delegate`  
+The conversion of an address-of method group to `delegate*` has roughly the same process as method group to `delegate`
 conversion. There are two additional restrictions to the existing process:
-- Only members of the method group that are marked as `static` will be considered. 
-- Only a `func*` with a managed calling convention can be the target of such a conversion.
 
-This means developers can depend on overload resolution rules to work in conjunction with the 
+- Only members of the method group that are marked as `static` will be considered.
+- Only a `delegate*` with a managed calling convention can be the target of such a conversion.
+
+This means developers can depend on overload resolution rules to work in conjunction with the
 address-of operator:
 
 ``` csharp
-unsafe class Util { 
-    public static void Log() { } 
-    public static void Log(string p1) { } 
+unsafe class Util {
+    public static void Log() { }
+    public static void Log(string p1) { }
     public static void Log(int i) { };
 
     void Use() {
-        func* void() a1 = &Log; // Log()
-        func* void(int) a2 = &Log; // Log(int i)
+        delegate*<void> a1 = &Log; // Log()
+        delegate*<int, void> a2 = &Log; // Log(int i)
 
         // Error: ambiguous conversion from method group Log to "void*"
-        void* v = &Log; 
+        void* v = &Log;
     }
 ```
 
 The address-of operator will be implemented using the `ldftn` instruction.
 
 Restrictions of this feature:
+
 - Only applies to methods marked as `static`.
 - Non-`static` local functions cannot be used in `&`. The implementation details of these methods are
-deliberately not specified by the language. This includes whether they are static vs. instance or
-exactly what signature they are emitted with.
+  deliberately not specified by the language. This includes whether they are static vs. instance or
+  exactly what signature they are emitted with.
 
 ### Better function member
+
 The better function member specification will be changed to include the following line:
 
-> A `func*` is more specific than `void*`
+> A `delegate*` is more specific than `void*`
 
-This means that it is possible to overload on `void*` and a `func*` and still sensibly use the address-of operator.
+This means that it is possible to overload on `void*` and a `delegate*` and still sensibly use the address-of operator.
```

Comment:
>sensibly [](start = 82, length = 8)

I don't see how.  You haven't defined a conversion from an "address of a method group" to `void*`.